src/components/home/AppBar.jsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Button from '@mui/material/Button';
+import AppBar from './AppBarLayout';
+import Toolbar from './ToolBar';
+
+const rightLink = {
+  fontSize: 16,
+  color: 'common.white',
+  ml: 3,
+};
+
+function AppAppBar() {
+  return (
+    <div>
+      <AppBar position="fixed" sx={{ bgcolor: '#1D97A7' }}>
```

Comment:
this bgcolor should be a variable, not hardcoded like this.
do the same where you have hardcoded colors.
make sure they are predefined